src/common/components/resetPasswordModal/resetPasswordModal.tpl.html

```diff
@@ -0,0 +1,12 @@
+<div class="col-md-12 col-xs-12">
+  <p>Enter your registered email address below and we'll send a new password to that email.</p>
+  <form>
+    <div class="form-group">
+      <label for="email">Email</label>
+      <input type="email" class="form-control form-control-subtle" placeholder="Email">
+    </div>
+    <button type="submit" class="btn btn-block btn-primary">Reset My Password</button>
+  </form>
+  <p class="mt- text-center">Never mind
```

Comment:
I feel like this needs a period or something after to separate it from the link. They are 2 different phrases.